cmd/prommetric/metric.go

```diff
@@ -0,0 +1,33 @@
+package prommetric
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type PromMeasure struct {
+	Temp int `json:"temp"`
+	Hum  int `json:"hum"`
+}
+
+var (
+	Temp = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "temperature_celsius",
+		Help: "The temperature of the hen house in celsius.",
+	})
+
+	Hum = prometheus.NewGauge(prometheus.GaugeOpts{
+		Name: "humidity_percent",
+		Help: "The humidity of the hen house in percent.",
+	})
+)
+
+func (m PromMeasure) SetTempAndHum() {
+	Temp.Set(float64(m.Temp / 10))
+	Hum.Set(float64(m.Hum / 10))
+
+	fmt.Printf("%s: SetTempAndHum set values: Temperature: %d °C, Humidity: %d %%\n",
+		time.Now().String(), m.Temp/10, m.Hum/10)
```

Comment:
time.Now() returns a time.Time, which implements fmt.Stringer -> you don't need to call String() here.
However - what is the format of Time.String() ?

How about this?
```suggestion
		time.Now().Format(time.RFC3339), m.Temp/10, m.Hum/10)
```

Also, we're not printing, here, the values that we set previously. We probably want to reuse them somehow - `Temp.Get()` ?